### PR TITLE
COMMERCE-1687 Add upgrade process for updating CPAttachment groupIds

### PR DESCRIPTION
**Pull request by alecsloan**
related to #1445

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test

File: commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v1_8_0/CPAttachmentFileEntryGroupIdUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * @author Alec Sloan
+ */
+public class CPAttachmentFileEntryGroupIdUpgradeProcess extends UpgradeProcess {
```
Comment (ethanbustad) [inline commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v1_8_0/CPAttachmentFileEntryGroupIdUpgradeProcess.java]: This should still be renamed to `CPAttachmentFileEntryUpgradeProcess` :)

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**